qiita_db/software.py

```diff
@@ -514,6 +516,22 @@ def start_script(self):
             qdb.sql_connection.TRN.add(sql, [self.id])
             return qdb.sql_connection.TRN.execute_fetchlast()
 
+    @property
+    def default_workflows(self):
+        """Returns the default workflows attached to the current software
+
+        Returns
+        -------
+        generator of qiita_db.software.DefaultWorkflow"""
```

Comment:
I don't I need to indent this because that line is exposing the type of the return - I've added another line with the description of the return.